packages/opentelemetry-instrumentation-xml-http-request/src/xhr.ts

```diff
@@ -216,10 +217,13 @@ export class XMLHttpRequestInstrumentation extends InstrumentationBase<XMLHttpRe
     xhrMem.createdResources = {
       observer: new PerformanceObserver(list => {
         const entries = list.getEntries() as PerformanceResourceTiming[];
+        const urlNormalizingAnchor = getUrlNormalizingAnchor();
+        urlNormalizingAnchor.href = spanUrl;
+
         entries.forEach(entry => {
           if (
             entry.initiatorType === 'xmlhttprequest' &&
-            entry.name === spanUrl
+            entry.name === urlNormalizingAnchor.href
```

Comment:
Note: This won't work with all IE versions (depending on how it's hosted) as can rewrite the URL as http://xxxx:443/yyy (which is just annoying).

For Application Insights we added a helper function that we run everything through to avoid the issue and how all URL's (page and dependency (XHR)) calls get reported. https://github.com/microsoft/ApplicationInsights-JS/blob/master/shared/AppInsightsCommon/src/UrlHelperFuncs.ts#L86-L93

Reply:
And the URL passed into the above function is generated by the same <a /> href trick being used here, we just "cache" a couple to avoid recreating new elements everytime.

Reply:
I'm not sure which versions of IE we support right now ? cc @obecny @dyladan

Reply:
Sorry was on vacation. Anyways IE doesn't have PerformanceObserver support so this should not be an issue here as this code is never ran in IE. But this .href trick is used elsewhere also so maybe lets create a separate issue to figure it out?

Reply:
@vmarchaud officially i'm not sure we claim any specific browser support. Unofficially, I think we only really work in IE 11

@MSNev is this a blocker for this PR?

Reply:
I don't believe it's a blocker, and it can be fixed later (if people complain). 

And it should be fixed more generally so that any checked / reported URLs don't get logged as http:///domain.xxx:443/ which is the main issue and only becomes a code bug when comparing different (unnormalized) Urls.